CHANGELOG.md

```diff
@@ -4,9 +4,11 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+##  - 2024-04-26
+
 ### Added
 
-- 
+- add dynamic asset paths
```

Comment:
Please remove the "add" in this line as the line is placed in the "Added" section already ;) Consider the section's headline "Added" as the start of the sentence.